Bottom: 7682ea3d21312d2f3a6e5598760f0f9bf13f22b2
Top:    3ea5984ce681bd21bef69f872ee119fffecdf4bf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-14 23:58:41 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
